### DISCUSSION TITLE
Adding a return link from figure images to figure callouts

### DISCUSSION THREAD

**Discussion by zsofiaj** [thegetty/quire#795] [Feedback]
Our editors and authors love the fact that users can click back between the main essay text and notes. Some users say they'd like to be able to do the same thing on the images. For example, if there’s a callout for fig. 2, you can click on that and it takes you to the image. We could imagine someone (a scholar) skimming the entry and looking at an image and wanting to be quickly taken to the section of the entry that talks about that image. Perhaps “Fig. 1” could be a hyperlink back to the figure callout. Or you could consider adding a little return symbol at the end of the caption, like at the end of each note. Just wanted to pass on this suggestion, but maybe this isn't possible.

**Comment by Erin-Cecele** [comment]
Thank you, @zsofiaj.  We really appreciate this type of feedback! 

I would love to hear the community's thoughts on this. Are there workarounds to achieve this? 
If not, is this something other people would like to see incorporated into Quire's capabilities? Are there any potential hurdles?
